ap_api/callbacks/fee/index.mdx

```diff
@@ -5,6 +5,8 @@ order: 0
 
 import {EndpointsTable} from "@site/src/components/EndpointsTable";
 
+The `/fee` is a special endpoint, that is called to fetch fee information for a transaction without a quote.
```

Comment:
Can we add some more context here? I framed this endpoint as optional in the user guide, in case you want to support clients who want to skip the quote creation process and are ok with using the businesses' market rate.